.github/workflows/build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: |
           mkdir -p build
           cargo about generate about.hbs > build/licenses.html
-      - uses: actions/upload-artifact@v3.1.2
+      - uses: actions/upload-artifact@v4
```

Comment:
I would recommend the exact stable version: v4.3.3. In the past we had some problems that the latest version was upgraded and our pipeline was crashing. This is why the documentation was extended about that: https://eclipse-ankaios.github.io/ankaios/0.3/development/ci-cd/#adding-a-new-github-action

Reply:
ok, used: for upload-artifact v4.3.3 and v4.1.6 for download-artifact